packages/lit-html/src/lit-html.ts

```diff
@@ -1432,6 +1432,21 @@ export type {EventPart};
 class EventPart extends AttributePart {
   readonly type = EVENT_PART;
 
+  constructor(
+    element: HTMLElement,
+    name: string,
+    strings: ReadonlyArray<string>,
+    parent: Disconnectable | undefined,
+    options: RenderOptions | undefined
+  ) {
+    super(element, name, strings, parent, options);
+
```

Comment:
@bicknellr can you see if you put the entire body of the constructor inside an `if (DEV_MODE)` conditional if Terser will eliminate the constructor completely?

Reply:
The constructor was apparently already here because it's used to assign `type`, but making this check depend on `DEV_MODE` also ended up converting its use of `...arguments` to into normal parameters:

![Screen Shot 2021-08-16 at 17 13 27](https://user-images.githubusercontent.com/406614/129644690-013ba6ea-e2db-4ab3-8c5a-a9fe80f0a30d.png)

Reply:
Or, rather, adding this constructor with its regular arguments caused that change and checking `DEV_MODE` here prevented this new block from being added.

Reply:
that seems ok... just a few more chars